ddl/util/syncer.go

```diff
@@ -388,3 +411,109 @@ func (s *schemaVersionSyncer) OwnerCheckAllVersions(ctx context.Context, latestV
 		time.Sleep(checkVersInterval)
 	}
 }
+
+const (
+	opDefaultRetryCnt     = 10
+	failedGetTTLLimit     = 20
+	checkCleanJobInterval = 1 * time.Second
+	opDefaultTimeout      = 3 * time.Second
+	opRetryInterval       = 500 * time.Millisecond
+)
+
+// NeededCleanTTL is exported for testing.
+var NeededCleanTTL = int64(-60)
+
+func (s *schemaVersionSyncer) StartCleanWork() {
+	for {
+		time.Sleep(checkCleanJobInterval)
```

Comment:
Why not use a timer to block on `select case` arm? We cannot execute `doCleanExpirePaths` in time if receive a task but in sleeping.

Reply:
Furthermore, the task dispatch thread will be blocked if there are two tasks be dispatched in one second.